src/common/types.h

```diff
@@ -20,6 +20,8 @@ enum class PSBTError {
     EXTERNAL_SIGNER_NOT_FOUND,
     EXTERNAL_SIGNER_FAILED,
     UNSUPPORTED,
+    INCOMPLETE,
+    OK,
```

Comment:
nit: Having an `OK` variant makes this more into an `PSBTResult` rather than `PSBTError`

Reply:
Will leave for a followup